Core/src/org/sleuthkit/autopsy/report/ReportProgressPanel.java

```diff
@@ -44,6 +47,12 @@
         ERROR
     }
 
+    public enum Events {
+
+        COMPLETED,
+        NOT_COMPLETED
```

Comment:
Only completed is used, so I'm just going to get rid of it. Don't know why I have it in the first place.